### DISCUSSION TITLE
Minimal setup list for web developers on a fresh macOS install | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#65] [Announcements]
# Minimal setup list for web developers on a fresh macOS install | Leo's dev blog – stories, insights, and ideas

Having a new Mac device is always exciting for every developer, but it can also be challenging, especially when it comes to setting up a new dev environment. Here is a list of the minimal setup you need to get started with web development on a fresh macOS install.

https://www.leohuynh.dev/blog/dev-setup-minimal-list-for-web-dev-on-a-fresh-macos-install

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡
If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
